examples/logger.py

```diff
@@ -133,6 +136,29 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         logger = logging.getLogger()
         logger.setLevel(logging.DEBUG)
         logger.addHandler(self.handler)
+        self.initLevelSelector(self.loggerFrame.levelSelector)
+
+    def initLevelSelector(self, box: QComboBox):
+        """Initializes a combo box for selecting level."""
+        box.addItem("DEBUG")
+        box.addItem("INFO")
+        box.addItem("WARNING")
+        box.addItem("ERROR")
+        box.addItem("CRITICAL")
+        box.activated[str].connect(self.fixLevel)
+
+    def fixLevel(self, text):
+        """Responds to the levelSelector widget and changes the handler's level."""
+        if text == "DEBUG":
```

Comment:
I suggest a simpler solution!

```python
self.handler.setLevel(getattr(logging, text))
```

Reply:
I used this method at first, but after reading @kangz12345 's suggestion I used dictionary instead.